### PR TITLE
Vulnerability Dashboard Documentation

### PR DESCRIPTION
**Pull request by alokshukla1978**

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -22,4 +22,7 @@
   * [Integrating Bamboo Builds](integrating-with-shiftleft/integrating-bamboo-builds.md)
   * [Integrating TeamCity Builds](integrating-with-shiftleft/integrating-teamcity-builds.md)
   * [Integrating Travis Builds](integrating-with-shiftleft/integrating-travis-builds.md)
+* Using Shiftleft in Security/Devlopment Workflows
```
Comment (vlad-alexandru-ionescu) [inline SUMMARY.md]:
```suggestion
* Using ShiftLeft in Security/Development Workflows
```

File: SUMMARY.md [hunk 2]
```diff
@@ -22,4 +22,7 @@
...
+  * [Vulnerability Dashboard](using-shiftleft/vulnerability-dashboard.md)
+  * Integrating with JIRA
```
Comment (vlad-alexandru-ionescu) [inline SUMMARY.md]: I wouldn't add this yet, since it doesn't yet lead anywhere.

File: using-shiftleft/vulnerability-dashboard.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Vulnerability Dashboard: Measure security quality of releases over time
+
+Vulnerability Dashboard is a singular view of application security quality metrics including a list of vulnerabilities based on static/runtime analysis of applications. Engineering and AppSec teams can use these metrics to measure progress of their security improvement over time.
+
+Vulnerability dashboard offers *three principal metrics*:
+
+## Progressive build (security) quality
+It is measured by the number of vulnerabilities fixed in the nth build vs n-1 build. In general, one could perform custom comparisions between any two builds. This helps measure progress (or decline) of security quality between releases.
+
+![Vulnerabilities - Release Trend](https://cdn-images-1.medium.com/max/800/0*0w7vzQlK6XsCH0-z)
```
Comment (vlad-alexandru-ionescu) [inline using-shiftleft/vulnerability-dashboard.md]: This image should be copied within the repository itself and then linked. You'll find examples on how to link repo images in other pages.

Comment (alokshukla1978) [reply using-shiftleft/vulnerability-dashboard.md]: Done

**Pull request merged**